setup/quickstart/halyard-local/index.md

```diff
@@ -0,0 +1,81 @@
+---
+layout: single
+title:  "Halyard on your local machine Quickstart"
```

Comment:
I would make this clear this is for development, e.g. `"Halyard for developing Spinnaker on your local machine Quickstart"`.

Reply:
Done.